src/App.tsx

```diff
@@ -36,6 +37,7 @@ export const App: React.FC<AppProps> = ({ args = [], options = {} }) => {
     <BrowserSessionProvider value={session}>
       {session ? (
         <>
+          <NowPlaying />
           <SongProgress />
           <UserControls />
         </>
```

Comment:
We should start thinking about how we want to do this. More specifically how we want to handle focus shifting between controls.